test/common/testlib.py

```diff
@@ -571,14 +600,21 @@ def snapshot(self, title, label=None):
             self.cdp.command("clearExceptions()")
 
             filename = "{0}-{1}.png".format(label or self.label, title)
-            ret = self.cdp.invoke("Page.captureScreenshot", no_trace=True)
-            if "data" in ret:
-                with open(filename, 'wb') as f:
-                    f.write(base64.standard_b64decode(ret["data"]))
-                attach(filename)
-                print("Wrote screenshot to " + filename)
-            else:
-                print("Screenshot not available")
+            if (self.cdp.browser == "chromium"):
+                ret = self.cdp.invoke("Page.captureScreenshot", no_trace=True)
```

Comment:
Is this API just not implemented, or buggy? In the latter case, is there a bug report for that?

Reply:
Not implemented. There is tracker anyway https://bugzilla.mozilla.org/show_bug.cgi?id=1549466